tests/core/data/all_config.toml

```diff
@@ -7,7 +7,7 @@ ny = 10
 
 [core.timing]
 start_date = "2020-01-01"
-update_interval = "10 minutes"
+update_interval = "2 weeks"
```

Comment:
Why did you chose 2 weeks instead of 1 months here?

Reply:
This file is used for testing, basically wanted something here that was different from the default value, which is 1 month. I basically made everything in this file different to their respective default values to check that overwriting of defaults works. I don't really need to do this for every single tag though, so could change this to a month if that makes things clearer

Reply:
I'd just add a comment to note the deliberate changes from default. Another advantage of TOML over JSON 😄